Add outreachy'22 wrap-up blog

ssurbhi560 opened the pull request:
Adding the outreachy wrap-up blog with a brief description of all the tickets that were solved during the internship.

ssurbhi560 commented:
Hi @jaimergp, the blog is done! :)

jaimergp commented on posts/2022-08-19-outreachy-wrap-up-blog-2022.rst:
```diff
@@ -0,0 +1,68 @@
+============================
+
+This blog is about my work during my Outreachy internship with `conda-forge <https://conda-forge.github.io>`__. Before that a little about me - 
+I am `Surbhi <https://github.com/ssurbhi560>`__, an outreachy intern with conda-forge for the May-August 2022 cohort and I worked on documenting the conda-forge ecosystem.
```
```suggestion
I am `Surbhi <https://github.com/ssurbhi560>`__, an Outreachy intern with conda-forge for the May-August 2022 cohort and I worked on documenting the conda-forge ecosystem.
```

jaimergp commented on posts/2022-08-19-outreachy-wrap-up-blog-2022.rst:
```diff
@@ -0,0 +1,68 @@
+
+10. `Add a section in docs on security aspects of conda-forge (#1808) <https://github.com/conda-forge/conda-forge.github.io/issues/1808>`__. Working on this `#1812 <https://github.com/conda-forge/conda-forge.github.io/pull/1812>`__.
+
+        Currently, the documentation regarding the security considerations of conda-forge builds is scattered throughout the documentation, and therefore it is hard to find and read it. With this issue, we will put all the information together in one place, which will help maintainers and users to know more about how conda-forge secures its packages.
```
```suggestion
        Currently, information regarding the security considerations of conda-forge builds is scattered throughout the documentation, and therefore it is hard to find and read. With this issue, we will put all the information together in one place, which will help maintainers and users to know more about how conda-forge secures its packages and infrastructure.
```

jaimergp reviewed (approved):
I fixed a little issue with the CI, and added a couple of (optional) suggestions for the text. LGTM!

Pinging @conda-forge/core and @kathatherine in case anybody else wants to take a look.

jaimergp commented on posts/2022-08-19-outreachy-wrap-up-blog-2022.rst:
```diff
@@ -0,0 +1,68 @@
+.. post:: 2022-08-19
+  :tags: Outreachy
+  :author: Surbhi Sharma
+  :redirect: 2022/08/19/outreachy-wrap-up-blog-2022
```
```suggestion
.. post:: 2022-08-26
  :tags: Outreachy
  :author: Surbhi Sharma
  :redirect: 2022/08/26/outreachy-wrap-up-blog-2022
```

Let's fix the date (assuming we merge tomorrow) on this metadata block _and_ the filename.

ssurbhi560 commented:
> I fixed a little issue with the CI, and added a couple of (optional) suggestions for the text. LGTM!

The commits Jaime made to fix the CI got deleted because of some mess I did with git. Sorry for that; I have pushed those changes again now.

kathatherine commented on posts/2022-08-19-outreachy-wrap-up-blog-2022.rst:
```diff
@@ -0,0 +1,68 @@
+
+        With this issue we added ​​packaging instructions for Perl packages with different build systems in the documentation.
+
+7.  `DOC: Update documentation about tokens (#1532) <https://github.com/conda-forge/conda-forge.github.io/issues/1532>`__. Working on this `#1793 <https://github.com/conda-forge/conda-forge.github.io/pull/1793>`__.
```
```suggestion
7.  `DOC: Update documentation about tokens (#1532) <https://github.com/conda-forge/conda-forge.github.io/issues/1532>`__. Closed with `#1793 <https://github.com/conda-forge/conda-forge.github.io/pull/1793>`__.
```

kathatherine commented on posts/2022-08-19-outreachy-wrap-up-blog-2022.rst:
```diff
@@ -0,0 +1,68 @@
+
+        With these, we added more documentation on migrations and migrators, which would help maintainers find answers to questions like - What is a migrator/migration, and what does it do? When can (and why would) they should reject a migration PR? And so on.
+
+10. `Add a section in docs on security aspects of conda-forge (#1808) <https://github.com/conda-forge/conda-forge.github.io/issues/1808>`__. Working on this `#1812 <https://github.com/conda-forge/conda-forge.github.io/pull/1812>`__.
```
```suggestion
10. `Add a section in docs on security aspects of conda-forge (#1808) <https://github.com/conda-forge/conda-forge.github.io/issues/1808>`__. Closed with `#1812 <https://github.com/conda-forge/conda-forge.github.io/pull/1812>`__.
```

kathatherine reviewed (changes requested):
Looks great! Just make sure those two issues that we closed in the last week are marked as such in the blog.

kathatherine reviewed (approved):
LGTM!

kathatherine commented:
@beckermr @jaimergp Shall we go ahead and merge this?

jakirkham commented:
Thanks Surbhi! Also thanks all for reviewing 🙏

jakirkham commented:
cc @tnabtaf (for awareness)

jakirkham commented:
Noticed a small formatting thing. Submitted PR ( https://github.com/conda-forge/blog/pull/58 ) to adjust that
